project/Project.scala

```diff
@@ -28,37 +28,38 @@ import com.typesafe.sbt.site.SphinxSupport.Sphinx
 object Zipkin extends Build {
   val zipkinVersion = "1.2.0-SNAPSHOT"
 
-  val finagleVersion = "6.26.0"
-  val utilVersion = "6.25.0"
-  val scroogeVersion = "3.19.0"
-  val zookeeperVersions = Map(
+  def finagle(name: String) = "com.twitter" %% ("finagle-" + name) % "6.26.0"
```

Comment:
Ah, that's right, I didn't think of that. Yeah, it's a little bit of magic we'd sprinkle all over the place. I'm fine with that too, don't have a strong opinion at this point.

For now I think let's leave it like this, if anyone feels strongly about changing this we can always go back and do it.